### PR TITLE
Change visuals of back buttons

### PR DESCRIPTION
**Pull request by wozniaczek123**

### CODE REVIEW

File: patches/arrows.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    
+    .close-button {
+        background-color: white !important;
+        box-shadow: none !important;
+        position: absolute !important;
+        top: calc((60px - 40px) / 2) !important;
+        cursor: pointer;
+        padding: 0 !important;
+        margin: 0 0 0 10px !important;
```
Comment (banocean) [inline patches/arrows.css:17]:
```suggestion
        margin: 0 0 0 10px !important;
        border: none !important;
```

Comment (banocean) [reply patches/arrows.css:17]: it fixes alignement

**Review by banocean — CHANGES_REQUESTED** [review]

**Comment by PanLiszka** [comment]
> image opera_FljnD2xc4M opera_mQDnCqJ2Bs
> 
> I think the text should be aligned 2.5px lower

opera_rj3BbM9Jyg

Setting `padding: 17.5px 48px` on .modal__title fixes the problem i guess

**Review by PanLiszka — APPROVED** [review]
LGTM!

**Review by rotkiW285 — APPROVED** [review]
LGTM

**Review by banocean — APPROVED** [review]

**Pull request merged**